cmake/RunTest.cmake

```diff
@@ -4,11 +4,11 @@
 # The CLEANUP_CMD command is run after the test is finished
 
 find_program(BASH bash)
-execute_process(COMMAND ${BASH} ${FILE_PATH}
+execute_process(COMMAND ${BASH} -e ${FILE_PATH}
     RESULT_VARIABLE test_result_${FILE_PATH}
     WORKING_DIRECTORY ${WORKING_DIRECTORY}
 )
-execute_process(COMMAND ${BASH} -c "${CLEANUP_CMD}"
```

Comment:
This to me looks like a duplication given https://github.com/MRtrix3/mrtrix3/pull/2865/commits/f9c27cdbf775251a500a656c84b76fa7dbbc1a2f#diff-53a4782583522f5aad0aa4173101cd9a0201a13f75ca5e2f159f94eaaa9a48baR55 includes the "`bash -c`" within the contents of `CLEANUP_CMD`; but this change was not itself sufficient to resolve the failure to delete temporary files.

Reply:
Notably, without this the log file would be filled with:
`/usr/bin/bash: line 1: /usr/bin/bash -c rm: No such file or directory`